app-modules/ai/src/Filament/Pages/ManageAiSettings.php

```diff
@@ -84,7 +84,7 @@ public static function canAccess(): bool
             return false;
         }
 
-        return $user->can(['product_admin.view-any']);
+        return $user->can(['saas_global_admin.view-any']);
```

Comment:
This should be checking if they are a global admin rather than a permission. There is a helper for this. `$user->isSuperAdmin()`